src/dashboard/root.tsx

```diff
@@ -4,15 +4,36 @@ import * as ReactDOM from 'react-dom'
 import { Pond } from '@actyx-contrib/react-pond'
 import { App } from './App'
 
-const onError = () => {
+let error: unknown = undefined
+const onError = (e: unknown) => {
+  error = e
   setTimeout(() => location.reload(), 2500)
 }
 
+function Loading() {
```

Comment:
I'd probably make the `Loading` component very simple and remove the timing. It's nice, but people will implement their own loading component anyway.

The important thing is the reload on error.

Reply:
![advanced-tutorial](https://user-images.githubusercontent.com/282765/123675421-faacc580-d842-11eb-8682-529d98e7c67c.gif)

I just tested it. Works fine, so I will merge this for now. Feel free to change this anytime though.